index.html

```diff
@@ -4,9 +4,20 @@
     <title>About Me</title>
     <link href="styles.css"
           rel="stylesheet"/>
+ <img src="http://www.pharmaceutical-technology.com/uploads/storefront/25492/images/226400/large/deltav.jpg" />
+    <header>
+    <nav class="primary-navigation">
+    <ul>
+      <li>Education</li>
+      <li>Work Experience</li>
+      <li>Fun Facts</li>
+      </nav>
+    <div class="clear"></div>
+  </header>
```

Comment:
This should all be inside `<body>` instead of `<head>`. Also, indentation is off a bit.